### DISCUSSION TITLE
Nextcloud AIO 10.0

### DISCUSSION THREAD

**Discussion by johnnyq** [nextcloud/all-in-one#5624] [Questions] [answered]
Hey just curious I noticed that Nextcloud AIO 10.0 is still in beta. Typically you have released them within 7 days of the beta but this one has been 9 days. No rush or anything I upgraded way early, but just curious.
Thanks for your time and hard work

**Comment by Zoey2936** [comment, marked as answer]
see: https://github.com/nextcloud/all-in-one/issues/5544 "wait a week and see if new bugs get reported and don't merge any PR into the main branch of this repo in the meantime! (until 26.11.)"

**Reply by johnnyq** [reply]
Thanks for clarifying @Zoey2936 sounds like i should have waited till all the bugs were worked out :roll_eyes:
